specs/deposits.md

```diff
@@ -100,46 +120,46 @@ tooling (such as wallets and block explorers).
 
 [create-nonce]: https://github.com/ethereum/execution-specs/blob/617903a8f8d7b50cf71bf1aa733c37897c8d75c1/src/ethereum/frontier/utils/address.py#L40
 
-## L1 Attributes Deposit
+## L1 Attributes Deposited Transaction
 
-[l1-attributes-deposit]: #l1-attributes-deposit
+[l1-attr-deposit]: #l1-attributes-deposited-transaction
 
-An [L1 attributes deposit] is a deposit transaction sent to the [L1 attributes predeploy][predeploy]
-contract.
+An [L1 attributes deposited transaction][g-l1-attr-deposit] is a deposit transaction sent to the [L1
+attributes predeployed contract][predeploy].
 
 This transaction MUST have the following values:
 
 1. `from` is `0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001` (the address of the
 [L1 Attributes depositor account][depositor-account])
-2. `to` is `0x4200000000000000000000000000000000000014` (the address of the L1 attributes predeploy
-   contract).
+2. `to` is `0x4200000000000000000000000000000000000014` (the address of the [L1 attributes predeployed
+   contract][predeploy]).
 3. `mint` is `0`
 4. `value` is `0`
 5. `gasLimit` is set to the maximum available.
-6. `data` is an [ABI] encoded call to the [L1 attributes predeploy][predeploy] contract's
+6. `data` is an [ABI] encoded call to the [L1 attributes predeployed contract][predeploy]'s
    `setL1BlockValues()` function with correct values associated with the corresponding L1 block (cf.
-   [reference implementation][l1-attrib-ref-implem]).
+   [reference implementation][l1-attr-ref-implem]).
 
-   <!-- TODO: Define how this account pays gas on these transactions. -->
+No gas is paid for L1 attributes deposited transactions.
```

Comment:
Maybe change this to say that "gas" is paid for on L1? Should be done when we transcribe the deposit fee design in the spec.

Reply:
Technically it's not paid for on L1 either, it's the rollup itself as system inserting the deposit. The rate-limit is simply the L1 block production.